be/src/exec/parquet/column_reader.cpp

```diff
@@ -634,9 +645,23 @@ class ListColumnReader : public ColumnReader {
         offsets[0] = 0;
         def_rep_to_offset(_field->level_info, def_levels, rep_levels, num_levels, &offsets[0], &is_nulls[0],
                           &num_offsets);
-        return Status::OK();
+
+        for (int i = 1; i <= num_offsets; i++) {
+            array_column->offsets_column()->append(offsets[i]);
```

Comment:
This is a cost operation cause there are too many function calls.
I think better to get offsets_column and assigned by offsets?